src/components/expanded-state/profile/ProfileNameInput.js

```diff
@@ -12,8 +13,9 @@ const NameInput = styled(Input).attrs({
   returnKeyType: 'done',
   size: 'big',
   spellCheck: false,
-  weight: android ? 'normal' : 'bold',
 })`
+  ${fontWithWidth(fonts.weight.bold)};
+  ${android ? 'height: 70; margin-vertical: -8' : ''};
```

Comment:
if `!android` it will print `;` which might end up in a parse error from styled-components